### PR TITLE
List contributors to P4Tools backend

### PR DESCRIPTION
**Pull request by jnfoster**

### CODE REVIEW

**Comment by jnfoster** [comment]
Tried to summarize the major contributors to P4Testgen. Please let me know if I missed someone, or if I got the history wrong.

**Review by pkotikal — APPROVED** [review]
Thank you!

File: backends/p4tools/CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+* @jnfoster (Cornell and Intel)
+
+# History
+
+Jed Liu was the original architect of P4Testgen and designed the abstract machine that allows using P4C's visitors to implement arbitrary traversals of the program IR. Building on Jed's design, Fabian Ruffy completed the symbolic interpreter for P4_16's core features and designed extensions for common open-source architectures. He also developed the model of packets, taint tracking, and concolic execution. Prathima Kotikalapudi developed the extensible test backend, with instantiations for PTF and STF. Vojtěch Havel assisted with the implementation of the symbolic interpreter and the development of P4Testgen extensions for several Intel architectures. Hanneli Tavante added support for strategies that guide the exploration of paths that maximize various notions of coverage. Oleksander Letychevskyi, Volodymyr Peschanenko, and Vlad Dubina developed the SMT-LIB interface to Z3, implemented front-end and mid-end passes such as parser unrolling, and completed the back end for the V1Model architecture on the BMv2 simple switch. Nate Foster guided the design of P4Testgen and contirbuted to various aspects of the symbolic interpreter.
```
Comment (fruffy) [inline backends/p4tools/CONTRIBUTORS.md:18]:
```suggestion
Jed Liu was the original architect of P4Testgen and designed the abstract machine that allows using P4C's visitors to implement arbitrary traversals of the program IR. Building on Jed's design, Fabian Ruffy completed the symbolic interpreter for P4_16's core features and designed extensions for common open-source architectures. He also developed the model of packets, taint tracking, and concolic execution. Prathima Kotikalapudi developed the extensible test backend, with instantiations for PTF and STF. Vojtěch Havel assisted with the implementation of the symbolic interpreter and the development of P4Testgen extensions for several Intel architectures. Hanneli Tavante added support for strategies that guide the exploration of paths that maximize various notions of coverage. Volodymyr Peschanenko, and Vlad Dubina developed the SMT-LIB interface to Z3, implemented front-end and mid-end passes such as parser unrolling, and completed the back end for the V1Model architecture on the BMv2 simple switch. Nate Foster guided the design of P4Testgen and contributed to various aspects of the symbolic interpreter.
```

File: backends/p4tools/CONTRIBUTORS.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Overview 
+
+P4Testgen is a test oracle for the P4 language. Given a P4_16 program and a specification of the underlying architecture, it automatically generates a comprehensive set of input/output tests that can be executed to validate a target device.
```
Comment (fruffy) [inline backends/p4tools/CONTRIBUTORS.md:3]: This is specific to P4Testgen, not P4Tools.

Comment (jnfoster) [reply backends/p4tools/CONTRIBUTORS.md:3]: I think that's fine for now...

**Review by fruffy — APPROVED** [review]

**Pull request merged**